src/main/java/org/team199/robot2020/subsystems/Feeder.java

```diff
@@ -16,34 +16,40 @@
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
 
 public class Feeder extends SubsystemBase {
-  private final WPI_TalonSRX beltMotor = MotorControllerFactory.createTalon(Constants.Feeder.kBeltMotor);
-  private final WPI_TalonSRX ejectMotor = MotorControllerFactory.createTalon(Constants.Feeder.kEjectMotor);
-  private final TimeOfFlight indexSensor = new TimeOfFlight(Constants.Feeder.kIndexSensor);
+  private static final double kBeltIntakeSpeed = .8;
+  private static final double kBeltEjectSpeed = 1;
+  private static final double kRollerEjectSpeed = 1;
+  private static final double kIndexerDistance = 127; // 5 inches in millimeters
```

Comment:
Perhaps make it inches times the factor so it can be changed easily?

Reply:
sounds good